webrender/src/device.rs

```diff
@@ -183,6 +183,11 @@ pub fn build_shader_strings(
     vs_source.push_str(gl_version_string);
     fs_source.push_str(gl_version_string);
 
+    // Insert the shader name to make debugging easier.
+    let name_string = format!("// {}\n", base_filename);
```

Comment:
perhaps, it would make sense to do in debug only?

Reply:
I use apitrace with release builds too and I don't think this adds any noticeable overhead so I lean towards leaving it in release builds as well unless you feel strongly about it.